test_regress/t/t_vpi_module.pl

```diff
@@ -18,7 +18,6 @@
     make_main => 0,
     make_pli => 1,
     iv_flags2 => ["-g2005-sv -D USE_VPI_NOT_DPI"],
-    v_flags2 => ["+define+USE_VPI_NOT_DPI"],
```

Comment:
I think the only simulator that needs to avoid DPI is Icarus.  And there are already ifdefs specifically for Icarus.

Reply:
Makes sense